common/types/envconfig.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+type EnvConfig struct {
+	PodNamespace    string `envconfig:"POD_NAMESPACE" default:"open-feature-operator-system"`
+	FlagdProxyImage string `envconfig:"FLAGD_PROXY_IMAGE" default:"ghcr.io/open-feature/flagd-proxy"`
+	// renovate: datasource=github-tags depName=open-feature/flagd/flagd-proxy
+	FlagdProxyTag            string `envconfig:"FLAGD_PROXY_TAG" default:"v0.3.0"`
+	FlagdProxyPort           int    `envconfig:"FLAGD_PROXY_PORT" default:"8015"`
+	FlagdProxyManagementPort int    `envconfig:"FLAGD_PROXY_MANAGEMENT_PORT" default:"8016"`
+	FlagdProxyDebugLogging   bool   `envconfig:"FLAGD_PROXY_DEBUG_LOGGING" default:"false"`
+
+	SidecarEnvVarPrefix   string `envconfig:"SIDECAR_ENV_VAR_PREFIX" default:"FLAGD"`
+	SidecarManagementPort int    `envconfig:"SIDECAR_MANAGEMENT_PORT" default:"8014"`
+	SidecarPort           int    `envconfig:"SIDECAR_PORT" default:"8013"`
+	SidecarImage          string `envconfig:"SIDECAR_IMAGE" default:"ghcr.io/open-feature/flagd"`
+	// renovate: datasource=github-tags depName=open-feature/flagd/flagd-proxy
```

Comment:
I think the sidecar should refer to flagd, not flagd-proxy

```suggestion
	// renovate: datasource=github-tags depName=open-feature/flagd/flagd
```

Reply:
good catch!